test/parallel/test-tls-getcipher.js

```diff
@@ -25,6 +25,7 @@ server.listen(common.PORT, '127.0.0.1', function() {
   var client = tls.connect({
     host: '127.0.0.1',
     port: common.PORT,
+    ciphers: cipher_list.join(':'),
```

Comment:
Ah, I think either way of solving is equal, but with leaving `RC4` we at least notice it when OpenSSL ever deprecates it.